tests/x11/firefox.pm

```diff
@@ -23,8 +23,9 @@ sub run() {
 
     $self->start_firefox;
     wait_still_screen;
-    send_key_until_needlematch('firefox-top-bar-highlighted', 'alt', 4, 10);
-    send_key('h');
+    send_key('alt');
```

Comment:
Yutao,
Can you please add some comments here why you change the hot key?
It can provide more clear info to reader :)

Reply:
I just change back. Original code is 'alt'